batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/validator/EmployeeValidator.java

```diff
@@ -0,0 +1,67 @@
+package com.lftechnology.batch7crud.validator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import javax.validation.ValidationException;
+
+import com.lftechnology.batch7crud.exception.ValidationExceptions;
+import com.lftechnology.batch7crud.model.Employee;
+
+public class EmployeeValidator implements Validator<Employee> {
+    @Override
+    public boolean isValid(Employee employee) throws ValidationException, ValidationExceptions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isValid" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)